src/murmur/Meta.h

```diff
@@ -192,6 +193,7 @@ class Meta : public QObject {
 		void bootAll();
 		bool boot(int);
 		bool banCheck(const QHostAddress &);
+		void successfulConnectionFrom(const QHostAddress &);
```

Comment:
Indentation seems to mismatch with the surrounding content

Reply:
Where do you see that? In my editor, git diff, and github it looks like it's at the same level...

Reply:
Oh lol. I'm currently using the GitHub android app and in there it was misaligned. It's no longer though so I guess that was a bug in the app. Sorry